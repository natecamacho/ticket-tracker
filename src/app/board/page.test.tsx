import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Board from "./page";

describe("Board component", () => {
  it("renders correctly", () => {
    render(<Board />);

    expect(screen.getByText("Team Board")).toBeInTheDocument();

    expect(screen.getByText("Up Next")).toBeInTheDocument();
    expect(screen.getByText("In Progress")).toBeInTheDocument();
    expect(screen.getByText("Done")).toBeInTheDocument();
  });

  it("renders initial Board component unchanged", () => {
    const { container } = render(<Board />);

    expect(container).toMatchSnapshot();
  });
});
