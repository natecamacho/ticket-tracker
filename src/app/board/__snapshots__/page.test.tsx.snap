// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board component renders initial Board component unchanged 1`] = `
<div>
  <div
    class="flex flex-col h-full"
  >
    Team Board
    <div
      class="flex flex-grow h-full"
    >
      <div
        class="flex w-80 h-12 bg-slate-200 mr-4 mt-4 items-center justify-center rounded-md"
      >
        Up Next
      </div>
      <div
        class="flex w-80 h-12 bg-slate-200 mr-4 mt-4 items-center justify-center rounded-md"
      >
        In Progress
      </div>
      <div
        class="flex w-80 h-12 bg-slate-200 mr-4 mt-4 items-center justify-center rounded-md"
      >
        Done
      </div>
    </div>
    <div
      class="flex flex-none h-full"
    >
      <div
        class="w-80 h-4/5 bg-slate-200 mr-4 mt-12 pt-2 text-center rounded-md"
      />
      <div
        class="w-80 h-4/5 bg-slate-200 mr-4 mt-12 pt-2 text-center rounded-md"
      />
      <div
        class="w-80 h-4/5 bg-slate-200 mr-4 mt-12 pt-2 text-center rounded-md"
      />
    </div>
  </div>
</div>
`;
